WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostPagerActivity.java

```diff
@@ -992,4 +1047,22 @@ public void onPositiveClicked(@NotNull String instanceTag) {
             fragment.onPositiveClicked(instanceTag);
         }
     }
+
+    @SuppressWarnings("unused")
+    @Subscribe(threadMode = ThreadMode.MAIN)
+    public void onPostUploaded(OnPostUploaded event) {
+        if (getLifecycle().getCurrentState().isAtLeast(STARTED)) {
+            int siteLocalId = AppPrefs.getSelectedSite();
```

Comment:
If we don't handle the OnPostUploaded event the Post uploaded snackbar with the VIEW button is never shown.
A similar pattern is used in WPMainActivity. The difference is that at WPMainActivity the register/unregister methods of the dispatcher are called in the onCreate/onDestroy. I thought of using the onResume/onPause methods since they were already used by the default eventbus.

Reply:
Hey @antonis quick clarification here; I was just referring to the following check

> if (getLifecycle().getCurrentState().isAtLeast(STARTED)) {

sorry I placed the comment in the wrong row 😊. I think the `WPMainActivity` is a bit different and the check is used as a filtering mechanism; do you think we need it here?

Reply:
You are right @develric . I misunderstood your comment.
I fixed this

Reply:
Thanks 👍! Minor np (neat pick): it would be good when fixing something (I mean in general not referring to this comment in particular) to reference the commit where you fixed it. You should be able to do this by copy/paste the commit hash in the comment. In this way who reviews can directly check the relevant code, thanks 🙇‍♂️.